app/screens/edit_profile/edit_profile.js

```diff
@@ -221,6 +235,13 @@ export default class EditProfile extends PureComponent {
         return RNFetchBlob.config(options).fetch('POST', `${Client4.getUserRoute(currentUser.id)}/image`, headers, [fileInfo]);
     };
 
+    removeProfileImage = async () => {
+        const {actions} = this.props;
+        const {currentUser} = this.props;
+        this.emitCanUpdateAccount(true);
```

Comment:
Is it really needed? You are already removing the image from the server, aren’t you?

Reply:
This is calling the action that then calls 'setDefaultProfileImage'. Is there another method to remove the image? It worked how I expected it would.

Reply:
Not another method.

I don't quite follow, will you remove the image once the user selects the option or will you remove the image when saving? cause this is to enable the save option as far as I remember.

I would expect that you remove the image then when tapping on save it actually removes it from the server, if you cancel this the current image should remain.

Thoughts?

Reply:
On second thought, I understand what you are asking now @enahum . It doesn't seem like I need to enable the Save button in the UI, which is what this event is triggering. So, you're right, I can take this out.